frontend/src/pages/AboutUs/AboutUsPage.tsx

```diff
@@ -56,26 +56,26 @@ export default function AboutUsPage() {
           make informed decisions, this scalable, feature-rich platform supports
           efforts to achieve health equity and justice for all.
         </p>
-        <h4 className='font-sansTitle font-medium'>
+        <h3 className='font-sansTitle font-medium'>
           Morehouse School of Medicine
-        </h4>
+        </h3>
         <p className='max-w-md px-6 text-left'>
           Morehouse School of Medicine (MSM), located in Atlanta, Ga., was
           founded in 1975 as the Medical Education Program at Morehouse College.
           In 1981, MSM became an independently chartered institution. MSM is
           among the nation’s leading educators of primary care physicians and
           was recently recognized as the top institution among U.S. medical
           schools for our social mission. We exist to: improve the health and
-          well-being of individuals and communities; increase the diversity of
-          the health professional and scientific workforce; and address primary
```

Comment:
I think we need these semicolons to distinguish the items after colon because the last one includes commas. It's still an awkward sentence but even harder to read without semicolons